app/controllers/gi_controller.rb

```diff
@@ -11,6 +11,12 @@ def client
   end
 
   def scrubbed_params
-    params.except(:action, :controller, :format)
+    params.except(:action, :controller, :format).transform_values do |v|
+      begin
+        v.respond_to?(:encode) && v.encode!('UTF-8', 'binary')
+      rescue StandardError
+        raise Common::Exceptions::InvalidFieldValue.new('parameter', v.scrub)
+      end
+    end
```

Comment:
would it be better to have a separate method altogether that does this rather than scrubbed params? One that is only used by search/autocomplete?